integration-job/provision_vm_and_start_kyma.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+set -o errexit
+
+cleanup() {
+    ARG=$?
+    echo "Removing instance kyma-integration-test-${RANDOM_ID}..."
+    gcloud compute instances delete kyma-integration-test-${RANDOM_ID}
+    exit $ARG
+}
+
+echo "Authenticating to Google Cloud..."
+export CLOUDSDK_COMPUTE_ZONE="europe-west3-a"
+export CLOUDSDK_COMPUTE_REGION="europe-west3"
+gcloud config set project kyma-project
+gcloud auth activate-service-account --key-file /var/run/secret/cloud.google.com/key.json
+
+RANDOM_ID=$(cat /dev/urandom | tr -dc 'a-z0-9' | fold -w 8 | head -n 1)
+
+echo "Creating a new instance named kyma-integration-test-${RANDOM_ID}..."
+gcloud compute instances create kyma-integration-test-${RANDOM_ID} \
+    --metadata enable-oslogin=TRUE --image debian-9-stretch-v20181009 \
+    --image-project debian-cloud --machine-type n1-standard-4 --boot-disk-size 20 \
+    --labels pull-number=$PULL_NUMBER,job-name=kyma-integration
```

Comment:
can PULL_NUMBER be null? In case of presubmit jobs probably not, but later this script can be used to run integration tests on the master branch, so we should handle also this situation.